elasticassignment/ageofempires/management/commands/create_indexes.py

```diff
@@ -0,0 +1,17 @@
+import ageofempires.models.civilization as Civilization
+import ageofempires.models.structure as Structure
+import ageofempires.models.technology as Technology
+import ageofempires.models.unit as Unit
+from django.core.management.base import BaseCommand
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **kwargs):
+        civ_intance = Civilization.Civilization()
+        struc_instance = Structure.Structure()
+        tech_instance = Technology.Technology()
+        uni_instance = Unit.Unit()
```

Comment:
- spelling mistake in `civ_intance`
- instead of using shorts, its better to use full names for objects. For example, `civilization` instead of `civ_instance`.
- To call constructor you don't need to do `ClassName.Classname()`. `ClassName()` works well.

Reply:
Throughout this PR, please check your variable naming and correct them:
> instead of using shorts, its better to use full names for objects. For example, civilization instead of civ_instance.